packages/shared/src/sdk-versioning/sdk-versions/javascript.json

```diff
@@ -33,15 +33,15 @@
     },
     {
       "version": "0.30.0",
-      "capabilities": ["visualEditorDragDrop"]
+      "capabilities": ["semverTargeting", "visualEditorDragDrop"]
```

Comment:
`semverTargeting` is only safe to use when the SDK version is `0.30.0` or above.

This is the original commit in `0.27.0` that crashes our app: https://github.com/growthbook/growthbook/pull/1286/files

And later it's fixed in `0.30.0`: https://github.com/growthbook/growthbook/pull/1286/files

Reply:
Hi @jdorn Thanks to the reply! 

As you said "as long as the attribute is passed into the SDK", but old version app has been publish to app stores and they can't pass this new attribute so they crashed (as screenshot is shown), that's why I found this is quite serious. From my understanding, the minimum version should still meet **everyone**'s basic needs, not just most of the users. Otherwise, it's still misleading, right?

In addition, I don't think updating the changelog can make a difference. Not every engineer will read changelog, they probably just trust this UI tip instead! 😂

But I respect your decision, if you still think this is not a risk I will close the PR.

Reply:
Dude, I think you miss the mobile case:
<img width="1513" alt="image" src="https://github.com/user-attachments/assets/f17dd18d-65fe-4376-993a-00a1f345408c">

React Native use `@growthbook/growthbook` under the hook, so this is not a edge case.